src/Nest/Mapping/Types/Specialized/Attachment/Attachment.cs

```diff
@@ -5,6 +5,13 @@
 
 namespace Nest
 {
+	/// <summary>
```

Comment:
`An indexed Attachement` feels like a SandCastle comment :cat: 

`<remarks>` should be used as extra documentation  about `why` something exists or a particular implementation inside the class was chosen. `<summary>` shows up in intellisense. Here I feel the remarks should be hoisted into the summary.